### ISSUE TITLE
Is DOCKER_HOST environment variable considered?

### ISSUE THREAD

**Issue by ilndinesh** [mviereck/x11docker#79]

**Comment by mviereck** [comment]
I am not sure about your question. You can set `DOCKER_HOST` before starting x11docker, and it will take effect.
E.g. `env DOCKER_HOST=mydockerhost x11docker [...]` 
Or:
```
export DOCKER_HOST=mydockerhost
x11docker [...]
```
If you encounter an issue, please provide a description. Should x11docker itself do something special?

**Comment by ilndinesh** [comment]
Thanks for more detail. If I encounter issue again, I will create another issue with more details and description.

**Comment by mviereck** [comment]
Depending on system setup x11docker runs docker with e.g. `su`, `sudo` or `pkexec`. (Compare option `--pw`). Maybe some environment variables get lost.
I just checked:
```
$ export DOCKER_HOST=xy
$ env | grep DOCKER
DOCKER_HOST=xy
$ sudo env | grep DOCKER
$
$ sudo -E env | grep DOCKER
DOCKER_HOST=xy
$ su -c env | grep DOCKER
Passwort: 
DOCKER_HOST=xy
$ su -c env | grep DOCKER
Passwort: 
DOCKER_HOST=xy
$ pkexec env | grep DOCKER
$
```
`sudo env` and `pkexec env` do not show `DOCKER_HOST`.

It seems x11docker should check for `DOCKER_HOST` and set it again.
Are there other important variables x11docker should regard?
I found some more `DOCKER_*` variables [here](https://stackify.com/docker-environment-variables/).

I will include a check for those variables in the next update.
